python/GafferUI/FileMenu.py

```diff
@@ -100,7 +100,7 @@ def __addScript( application, fileName, dialogueParentWindow = None, asNew = Fal
 
 	recoveryFileName = None
 	backups = GafferUI.Backups.acquire( application, createIfNecessary = False )
-	if backups is not None :
+	if backups is not None and "DISABLE_BACKUP" not in os.environ :
```

Comment:
Please remove this commit - this is the original approach, and doesn't contribute to the final result in this PR.